CHANGELOG.md

```diff
@@ -1,11 +1,24 @@
 # Changelog
 
-## [v1.22.0-rc](https://github.com/googleforgames/agones/tree/v1.22.0-rc) (2022-03-22)
+## [v1.22.0](https://github.com/googleforgames/agones/tree/v1.22.0) (2022-04-04)
+
+[Full Changelog](https://github.com/googleforgames/agones/compare/v1.22.0-rc...v1.22.0)
+
+**Fixed bugs:**
```

Comment:
I don't see https://github.com/googleforgames/agones/pull/2528 in this list, but it was a bug specifically fixed on the release candidate for the release. 

It's tagged with the 1.22 milestone, so I would expect it to show up here. Did you generate the release notes after it merged? I wonder if because we accidentally merged it into the wrong branch it isn't being picked up by the release notes tool.

Reply:
> I wonder if because we accidentally merged it into the wrong branch it isn't being picked up by the release notes tool.

I wouldn't entirely be shocked if it didn't get picked up, but @SaitejaTamma can you confirm that you ran the changelist notes tool after we merged the PR?

Reply:
Yes i did after the PR Merged